packages/components/pie-tag/src/tag.scss

```diff
@@ -29,9 +26,12 @@
     align-items: center;
     justify-content: center;
     gap: var(--dt-spacing-a);
-    height: var(--tag-height);
-    padding: var(--tag-padding);
-    border: none;
+    padding-inline-start: var(--tag-padding-inline);
```

Comment:
I think you can just do `padding: var(--tag-padding-block) var(--tag-padding-inline);` here

Reply:
done